examples/index.json

```diff
@@ -1878,6 +1878,14 @@
                 "verifyOutput": false,
                 "disablePlayground": true,
                 "isLearnByExample": false
+            },
+            {
```

Comment:
Shall we add this to a separate section, maybe `GraphQL Client`? This is how we structured the REST service and Client.

Reply:
@ThisaruGuruge In that case the new section will only contain 2 BBEs (client BBE and IDL client BBE). Will that be okay?

Reply:
@ThisaruGuruge we have a separate BBE for `Client`. Should we move that as well to the new `GraphQL Client` section?

Reply:
I think that would be fine. @praneesha WDYT?

Reply:
@praneesha WDYT?

Reply:
Also, in this case, we have to rename the `GraphQL` section to `GraphQL Service` (or `Services` whatever the conversion we use).

Reply:
@ThisaruGuruge @praneesha Updated with the changes